packages/sdk-codegen/src/python.gen.ts

```diff
@@ -441,11 +481,12 @@ ${this.hooks.join('\n')}
     const attrs: string[] = []
     const isEnum = type instanceof EnumType
     const baseClass = isEnum ? 'enum.Enum' : 'model.Model'
-    let attrsArgs = 'auto_attribs=True, kw_only=True'
     let usesReservedPythonKeyword = false
 
     if (!isEnum) {
-      for (const prop of Object.values(type.properties)) {
+      for (const prop of Object.values(type.properties).sort(
+        this.sortProperties
+      )) {
```

Comment:
We should not be sorting properties of a type.

Reply:
let's talk about it - I can try to find another solution for python but that might be a bit of work